pkg/cmd/cli/cli.go

```diff
@@ -292,7 +292,7 @@ func CommandFor(basename string) *cobra.Command {
 	case "kubectl":
 		cmd = NewCmdKubectl(basename, out)
 	default:
-		cmd = NewCommandCLI(basename, basename, in, out, errout)
+		cmd = NewCommandCLI("oc", "oc", in, out, errout)
```

Comment:
You're not right. This is the default entry for the `oc` series command line utils (currently, `oc` only). The entries used by `openshift` and `oadm` series are not here.